src/electionguard/chaum_pedersen.cpp

```diff
@@ -228,47 +227,41 @@ namespace electionguard
         auto inBounds_v0 = v0.isInBounds();
         auto inBounds_v1 = v1.isInBounds();
 
+        // c = H(Q,K,α,β,a0,b0,a1,b1)
```

Comment:
this is where most of the actualy "work" is captured in this PR that isnt just renaming